packages/transformers/js/core/src/hoist.rs

```diff
@@ -1196,6 +1226,58 @@ impl Collect {
   }
 }
 
+impl From<Collect> for CollectResult {
+  fn from(collect: Collect) -> CollectResult {
+    CollectResult {
+      imports: collect
+        .imports
+        .into_iter()
+        .map(
+          |(
+            local,
+            Import {
+              source,
+              specifier,
+              loc,
+              kind,
+            },
+          )| CollectImportedSymbol {
+            source,
+            local: local.0,
+            imported: specifier,
+            loc,
+            kind,
+          },
+        )
+        .collect(),
+      exports: collect
+        .exports
+        .into_iter()
+        .map(
```

Comment:
What's the reason to map these to a different struct with (almost) the same shape? Should we just expose the original? The less copying the better...

Reply:
That would be better, but they all contain `IdentId`  (so `(JsWord, SyntaxContext)`) and the letter prevents serialization.

Reply:
Ah... ok